RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -354,7 +354,12 @@ void finalizeMaximize(boolean speculative)
 		addBonusToMaximize($item[Cursed Magnifying Glass], 1000);
 	}
 	addBonusToMaximize($item[mafia thumb ring], 200); // adventures
+	if(get_property("auto_MLSafetyLimit").to_int() >= 25)
+	{
+		addBonusToMaximize($item[carnivorous potted plant], 200); // free kills
+	}
 	addBonusToMaximize($item[Mr. Screege\'s spectacles], 100); // meat stuff
+	addBonusToMaximize($item[can of mixed everything], 100); // random stuff
```

Comment:
forcing this is bad too. yes it can drop stuff. but we have other things we want to put in that slot. this is a very bad way of implementing it and we are better off just not doing it

Reply:
```suggestion
```